anitya/lib/backends/githubapi.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+
+"""
+ (c) 2014 - Copyright Red Hat Inc
+
+ Authors:
+   Michael Lutonsky <m@luto.at>
+
+"""
```

Comment:
I noticed that the tests had a license declaration. Should we put one of those here too?